### DISCUSSION TITLE
Enabling Positional Cost

### DISCUSSION THREAD

**Discussion by MdRanaSarkar** [mikel-brostrom/yolo_tracking#461] [Q&A]
Calculating cost matrix we use lambda to weight between positional cost and appearance cost.  As the lambda value set to 0 the positional cost is not taken in consideration to the actual cost matrix. 

Why the position cost omitted from the actual cost matrix ?

in tracker.py  line number 130 . 

![image](https://user-images.githubusercontent.com/37153510/179907127-4de0c228-2368-4200-90b2-356a0191a8dc.png)

**Comment by mikel-brostrom** [comment]
Sorry this is the old matching strategy, it should have been deleted when moving from DeepSORT to StrongSORT but was not. You can find the new matching here: https://github.com/mikel-brostrom/Yolov5_StrongSORT_OSNet/blob/07ae91cc30f03072cf702206e88053185d144eaa/strong_sort/sort/linear_assignment.py#L131

**Reply by MdRanaSarkar** [reply]
Thanks for your instant reply . Is this matching for multiclass or just single class ?

**Reply by mikel-brostrom** [reply]
Multi, all objects end up in this cost matrix
